crates/runtime/src/flight/do_put.rs

```diff
@@ -166,6 +166,7 @@ fn create_response_stream(
     first_message: &FlightData,
 ) -> impl futures::Stream<Item = Result<PutResult, Status>> + use<> {
     let dictionaries_by_id = Arc::new(HashMap::new());
+    tracing::debug!("Starting writing data to path: {path}");
```

Comment:
Message could be more descriptive.

Reply:
Information about the Flight method is included by default as part of the tracing (`DEBUG runtime::flight::do_put: `prefix), so updated the message from `Starting writing data to path: {path}` to `Starting writing data to dataset: {path}`